tests/test_yaml_parser.py

```diff
@@ -146,3 +146,13 @@ def test_invalid_empty_envs(tmp_path):
     with pytest.raises(ValueError) as e:
         Task.from_yaml(config_path)
     assert 'Environment variable \'env_key2\' is None.' in e.value.args[0]
+
```

Comment:
You may want to run `./format.sh` (after running `pip install -r requirements-dev.txt`) to keep the [linter](https://github.com/skypilot-org/skypilot/actions/runs/10126810572/job/28003876019?pr=3792) happy.

```suggestion


```